src/test/java/com/google/sps/UserTest.java

```diff
@@ -0,0 +1,89 @@
+package com.google.sps;
+
+import com.google.sps.user.User;
+import org.junit.Assert;
```

Comment:
We can import assertEquals so that the code can be shorter :)

Reply:
Done